test/unit/chrome_test.rb

```diff
@@ -45,22 +45,6 @@ class ChromeTest < Minitest::Test
     assert browser.platform.chrome_os?
   end
 
-  test "detects yandex browser" do
-    browser = Browser.new(Browser["YANDEX_BROWSER"])
-
-    assert browser.yandex?
-    assert browser.chrome?
-    refute browser.safari?
-    assert browser.webkit?
-    assert_equal "41.0.2272.118", browser.full_version
```

Comment:
This takes the underlying Chrome version, as opposed to the Yandex version (which makes sense if we are considering the browser to be chrome).